core/src/main/java/org/web3j/protocol/core/methods/response/TransactionReceipt.java

```diff
@@ -158,10 +158,10 @@ public void setStatus(String status) {
     }
 
     public boolean isStatusOK() {
-        if (null == status) {
+        if (null == getStatus()) {
```

Comment:
this is also in another PR - either remove it from here or add a test for this

Reply:
Changed various things to get CI to pass in the last few mins, however unclear as to how we'd test this change specifically - just redirects `isStatusOK()` to potentially use overridden method if it exists in child classes. Not an ideal situation, but equally nicer than having to basically entirely restructure control flow & transaction class structure as we previously tried to to fix this.

Reply:
you would have a mock and verify the method was correctly delegated

Reply:
Done